http-server/src/routes/alin_test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * for example:
+ * http://localhost:3000/alin?time=12:30&volume=5
+ */
+const os = require('os')
+const method = 'GET'
+const route = '/alin'
+var list = [ ];
+const handler = function handler (req, res) {
+  if (!req.query || !req.query.time || !req.query.volume) {
+    res.statusCode = 400
+    res.statusMessage = 'Bad request'
+    return res.end('Bad request')
+  } 
+  //const a = parseFloat(req.query.a)
+  //const b = parseFloat(req.query.b)
+  //const ack = `${a + b} from  ${os.type()}`
+  var obj = {
+    time: req.query.time,
+    volume: req.query.volume
+  };
+  list.push(obj);
+  for(var i = 0; i < list.length; i++) {

```

Comment:
If you want to inspect the list array, use console with for loop is not a good idea.
How about we remove the loop, inspect the array with break points in debug mode ?